rdrf/rdrf/settings.py

```diff
@@ -24,6 +24,9 @@
 SECURE_CONTENT_TYPE_NOSNIFF = env.get("secure_content_type_nosniff", PRODUCTION)
 SECURE_BROWSER_XSS_FILTER = env.get("secure_browser_xss_filter", PRODUCTION)
 SECURE_REDIRECT_EXEMPT = env.getlist("secure_redirect_exempt", [])
+SECURE_HSTS_SECONDS = 3600 # TODO: Bump to 1 week, 1 month, 1 year
```

Comment:
`SecurityMiddleware` only sets `Strict-Transport-Security` on https requests, so this doesn't affect dev.

That timeline is the generic recommendation, I think we can increase the length more quickly once we confirm it works.